docker-compose.yml

```diff
@@ -0,0 +1,22 @@
+version: '3.3'
```

Comment:
Like the Python version, please not Docker version in Github Wiki in setup page

Reply:
docker-compose 

- [ ]